src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -131,9 +127,11 @@ private NeoModule makeSwerveModule(int driveId, int steerId) {
   }
 
   private void configureBindings() {
-    new Trigger(m_driveController::getStartButton)
-        .onTrue(new InstantCommand(() -> m_chassis.resetPose(new Pose2d())));
-    new Trigger(m_driveController::getRightBumper)
+    m_driveController
+        .start()
+        .onTrue(new InstantCommand(() -> m_poseEstimator.resetPose(new Pose2d())));
```

Comment:
Let's turn this into a command factory in the pose estimator class. You'll have to use `Command.runOnce` since it's not a subsystem

Reply:
I don't fully understand what you mean by Command.runOnce. Is it similar to an InstantCommand?